noir-projects/aztec-nr/aztec/src/note/note_getter/mod.nr

```diff
@@ -37,15 +38,6 @@ fn extract_property_value_from_selector<let N: u32>(
     value_field
 }
 
-fn check_note_header<Note, let N: u32>(context: PrivateContext, note: Note)
-where
-    Note: NoteInterface<N>,
-{
-    let header = note.get_header();
-    let contract_address = context.this_address();
-    assert(header.contract_address.eq(contract_address), "Mismatch note header contract address.");
```

Comment:
Now I instead just directly check the address in the relevant places so there is no real change here.